app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -4,6 +4,7 @@ class OmniauthCallbacksController < Devise::OmniauthCallbacksController
   def google_oauth2
     if user.present?
       user.accept_invitation! if invitation_token
+      AvatarUploader.set_avatar_from_url!(user, google_avatar) if google_avatar
```

Comment:
So the `Method google_oauth2 has a Cognitive Complexity of 6 (exceeds 5 allowed).` error from Code Climate is being raised because this line is introducing more complexity in terms of conditionals. 

I think we could refactor this method to simplify the logic a bit!

```
def google_oauth2
  if user.present?
    user.accept_invitation! if invitation_token
    AvatarUploader.set_avatar_from_url!(user, google_avatar) if google_avatar

    flash[:notice] = I18n.t('devise.omniauth_callbacks.success',
                              kind: t('omniauth.google'))
    sign_in_and_redirect @user, event: :authentication
  end
  redirect_to new_user_session_path, notice: t('omniauth.access_denied')
end
```

Reply:
Thanks for the suggestion @julianguyen!

@bmwachajr and I are discussing the possibility of using the `upload` method in `cloundinary_service.rb` instead, but I will keep this in mind if we continue down the route of using the method in `AvatarUploader`.